### PR TITLE
Card validation

### PR DESCRIPTION
**Pull request by manideepk90**

### CODE REVIEW

**Review by aashu331998 — APPROVED** [review]

File: src/utility/reusableCodeFromWeb/CardPattern.res [hunk 1]
```diff
@@ -47,8 +47,9 @@ let cardPatterns = [
   {
     issuer: "Discover",
     pattern: %re("/^(6011|64[4-9]|65|622126|622[1-9][0-9][0-9]|6229[0-1][0-9]|622925)/"),
+    // pattern: %re("/^(6011|65|64[4-9]|622)/"),
     cvcLength: [3],
-    length: [16, 17, 18, 19],
+    length: [16],
```
Comment (seekshiva) [inline src/utility/reusableCodeFromWeb/CardPattern.res:52]: why this reduction? was the old info that Discover has cards with 17.18,19 digits a wrong info?

**Comment by manideepk90** [comment]
I downloaded this from the Discover Global Network and got the same information from the Juspay Indian payment team. They use a simple card validator to validate Discover cards, which have only 16 digits.

**Comment by manideepk90** [comment]
https://www.discoverglobalnetwork.com/content/dam/discover/en_us/dgn/docs/card-identification-features.pdf

File: src/utility/reusableCodeFromWeb/CardPattern.res (hunk 1 above)
Comment (manideepk90) [reply src/utility/reusableCodeFromWeb/CardPattern.res:52]: i got the information from [discover](https://www.discoverglobalnetwork.com/content/dam/discover/en_us/dgn/docs/card-identification-features.pdf) this includes that discover supports 16-digit card and i confirm with the payments page team. they are using 16 digit for discover card. so i modified with that information

**Pull request merged**